invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -170,10 +170,9 @@ const mapFormFields = (obj, parentField, isCreate, formFieldsConfig, dropDumpOnl
     const rows = formFieldsConfig[fieldName]?.rows || fieldSchema?.metadata?.rows;
     if (fieldSchema.type === "string" && rows) {
       return (
-        <TextArea
+        <RichInput
           key={fieldProps.fieldPath}
           fieldPath={fieldProps.fieldPath}
-          rows={rows}
```

Comment:
Have you checked if the `rows` prop is used or set somewhere? If yes, we lose functionality of defining a height of the field.

Reply:
the default height of editor is being used which can be overriden through editorConfig `min_height`  but that would not correspond with the scale of numeric value of rows, which rn is 10 and that would render a very small text area, I added a multiplication factor to that corresponds with the number of rows